console/my-account/reference-content/products-availability.mdx

```diff
@@ -0,0 +1,52 @@
+| **Compute**           | Instances                             | PAR1, PAR2, PAR3       | AMS1, AMS2              | WAW1, WAW2             |
+|                       | GPU                                   | PAR1, PAR2             | Not available yet       | Not available yet      |
+| **Bare Metal**        | Elastic Metal                         | PAR1, PAR2             | Not available yet       | Not available yet      |
+|                       | Apple Silicon                         | PAR3                   | Not available yet       | Not available yet      |
+|                       | Dedibox                               | DC2, DC3, DC5*         | AMS1                    | Not available yet      |
+| **Containers**        | Kubernetes                            | PAR1, PAR2, PAR3       | AMS1, AMS2              | WAW1, WAW2             |
+|                       | Container Registry                    | Across Paris region    | Across Amsterdam region | Across Warsaw region   |
+| **Storage**           | Object Storage Multi-AZ + One Zone IA | Across Paris region    | Across Amsterdam region | Across Warsaw region   |
+|                       | Object Storage Glacier                | Paris**                | Not available yet       | Not available yet      |
+|                       | Block Storage                         | PAR1, PAR2, PAR3       | AMS1, AMS2              | WAW1, WAW2             |
+| **Managed Databases** | PostgreSQL - MySQL                    | Across Paris region    | Across Amsterdam region | Across Warsaw region   |
+|                       | Redis                                 | PAR1, PAR2             | AMS1, AMS2              | WAW1                   |
+| **Network**           | Private Networks                      | PAR1, PAR2, PAR3       | AMS1, AMS2              | WAW1, WAW2             |
+|                       | Public Gateway                        | PAR1, PAR2             | AMS1, AMS2              | WAW1, WAW2             |
+|                       | Load Balancer                         | PAR1, PAR2             | AMS1, AMS2              | WAW1, WAW2             |
+|                       | Domains and DNS                       | Automatically assigned | Automatically assigned  | Automatically assigned |
+|                       | Functions                             | Across Paris region    | Across Amsterdam region | Across Warsaw region   |
+| **Serverless**        | Containers                            | Across Paris region    | Across Amsterdam region | Across Warsaw region   |
+|                       | Messaging & Queing                    | Automatically assigned | Automatically assigned  | Automatically assigned |
+|                       | TEM                                   | Automatically assigned | Automatically assigned  | Automatically assigned |
```

Comment:
I think it would be better to replace "TEM" with "Transactional Email", as it is always referred to as such :)